### ISSUE TITLE
The program preview error

### ISSUE THREAD

**Issue by walter8729** [Niek/obs-web#219]
Sorry my english is bad. The issue is that the program preview should not be change or update when I toggle on/off any scene sources, but in the the web program preview is changed, but in obs program program preview is maintain last changes after transition.

its working fine when i change to other scenes to preview other scenes or scenes items, but when i change or update scene items of the same scene "onAir" is when the error happens.
The program preview in studio mode, i think should be mantain inmutable untill press transition button
![OBS RE](https://user-images.githubusercontent.com/98748321/233792178-da24193f-7370-426e-aebf-5b9d277b2a1c.png)

**Comment by Niek** [comment]
I don't understand the issue completely. So you are in studio mode, toggle a source off, and what goes wrong?

**Comment by Niek** [comment]
Ah, I think I see what you mean: if both the preview and program scene are the same scene, and you toggle a source off, the preview in OBS-web shows the correct screenshot but the program scene is wrong. I need to check if this is an issue with OBS-web or with obs-websocket itself.

**Comment by Niek** [comment]
Looking at https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#getsourcescreenshot, I don't think there's a way to distinguish screenshots from a program vs preview scene. Can you open a feature request upstream? Once that is implemented, we can add support for it in OBS-web.

**Comment by walter8729** [comment]
You're right, I was reading the obs-websocket documentation, and you can only preview inputs, that is, a scene or an item from a scene. It would be necessary to implement functions like getProgramScreenShot(), or getPreviewScreenShot(), which return the previews of these outputs.
